Lua cmd struct member missing

## Current Behavior

<!--- What went wrong? -->

Error while compiling Lua files

## Steps to Reproduce

Compile Master Branch with VERBOS BEBUG enabled

<!-- Add relevant code and/or a live example -->

this log macro tries to access a missing struct member on cmd
DBGVLN("sending response for [%s] chunk=%u step=%u", cmd->name, nextStatusChunk, cmd->step);

<!-- Add stack traces -->

In file included from lib/LUA/lua.cpp:5:0:
lib/LUA/lua.cpp: In function 'void pushResponseChunk(luaItem_command*)':
lib/LUA/lua.cpp:155:61: error: 'struct luaItem_command' has no member named 'name'
   DBGVLN("sending response for [%s] chunk=%u step=%u", cmd->name, nextStatusChunk, cmd->step);
                                                             ^
lib/logging/logging.h:47:26: note: in definition of macro 'DBGLN'
       debugPrintf(msg, ##__VA_ARGS__); \
                          ^
lib/LUA/lua.cpp:155:3: note: in expansion of macro 'DBGVLN'
   DBGVLN("sending response for [%s] chunk=%u step=%u", cmd->name, nextStatusChunk, cmd->step);
   ^
*** [.pio\build\EXPRESS_TX\lib461\LUA\lua.cpp.o] Error 1
1.

2.

3.

4.

## Possible Solution (Not obligatory)

<!--- Suggest a reason for the bug or how to fix it. -->

remove the struct access, Not sure if the cmd ->name is needed, Was this struct member got removed by a mistake ?

## Details

<!--- Additional details you think might be relevant -->

<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Did you make any changes to the boilerplate after cloning it? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

Windows

- TX hardware:
- RX hardware:
- Handset model:
- OpenTX version (including nightly number)
- ExpressLRS version (TX & RX MUST MATCH):
- Packet Rate:
- Telemetry Ratio:
- user_defines:

--- a/src/lib/LUA/lua.cpp
+++ b/src/lib/LUA/lua.cpp
@@ -152,7 +152,7 @@ static uint8_t sendCRSFparam(crsf_frame_type_e frameType, uint8_t fieldChunk, st
 }
 
 static void pushResponseChunk(struct luaItem_command *cmd) {
-  DBGVLN("sending response for [%s] chunk=%u step=%u", cmd->name, nextStatusChunk, cmd->step);
+  DBGVLN("sending response for [%s] chunk=%u step=%u", cmd->common.name, nextStatusChunk, cmd->step);
   if (sendCRSFparam(CRSF_FRAMETYPE_PARAMETER_SETTINGS_ENTRY,nextStatusChunk,(struct luaPropertiesCommon *)cmd) == 0) {
     nextStatusChunk = 0;
   } else {

